app/assets/src/components/ui/controls/dropdowns/MultipleNestedDropdown.jsx

```diff
@@ -239,6 +239,7 @@ class MultipleNestedDropdown extends React.Component {
         label={labelText}
         value={label}
         rounded={this.props.rounded}
+        hideBadgeIfInsufficientSpace={this.props.hideBadgeIfInsufficientSpace}
```

Comment:
Make sure to add `hideBadgeIfInsufficientSpace` to the propTypes of all the relevant components.